products/rancher.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Rancher
+category: service
+tags: suse rancher kubernetes
+iconSlug: rancher
+permalink: /rancher
+alternate_urls:
+-   /suse-rancher
+releasePolicyLink: https://github.com/rancher/rancher/releases
```

Comment:
> this should be an official rancher site instead of the github repo, i guess?

Yes. A documentation on GitHub is OK, but it should explain how Rancher is released and supported.

Reply:
changed the link to the suse rancher lifecycle page